release_templates/vintage/aws/v1alpha3/cluster/awscluster.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: infrastructure.giantswarm.io/v1alpha3
+kind: AWSCluster
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/ui-api/management-api/crd/awsclusters.infrastructure.giantswarm.io/
+  labels:
+    cluster.x-k8s.io/cluster-name: ${cluster_id}
+    giantswarm.io/cluster: ${cluster_id}
+    giantswarm.io/organization: ${organization}
+    release.giantswarm.io/version: ${release}
+  name: ${cluster_id}
+  namespace: org-${organization}
+spec:
+  cluster:
+    description: ""
```

Comment:
Why is this one empty but the other one  (AWSMachineDeployment) has a default string?

Also we should assess how many of these values are actually ok to have - maybe cross checking with Phoenix?